src/pages/tag.js

```diff
@@ -74,6 +74,20 @@ export default class TagGridPage extends Component {
     this.props.uiStore.setSortBy(sortBy);
   };
 
+  renderHeaderMetadata = (metafields) => {
+    const metadata = [];
+    metafields.forEach((field) => {
+      if (field.valueType && field.valueType === "header") {
```

Comment:
`namespace` seems like a better fit for types like `header`, `custom` etc, where `valueType` would be better for `string`, `boolean`, `number` in cases where the meta value shouldn't be treated as a string.

Also, `header` should be `head` or `metatag` as to not be confused with the `<header>` tag, or a block on the page that's considered the header.